front/app/components/CustomFieldsForm/Fields/LinearScale/Labels.tsx

```diff
@@ -45,9 +48,13 @@ const Labels = ({ question, maximum }: Props) => {
         </>
       ) : (
         // Show labels as list underneath the buttons when more than 3 labels OR on mobile devices
-        <Box maxWidth={'100%'}>
+        <Box maxWidth={'100%'} role="list">
           {labelsFromSchema.map((label, index) => (
-            <Box display="flex" key={`${question.key}-${index}`}>
+            <Box
+              display="flex"
+              key={`${question.key}-${index}`}
+              role="listitem"
+            >
```

Comment:
Why wouldn't we use HTML instead of ARIA-attributes? :)

```
<Box as="ol" p="8px" maxWidth={'100%'}>
  {labelsFromSchema.map((label, index) => (
     <Box display="flex" key={`${question.key}-${index}`} as="li">
     ...
```

Reply:
Because this is an easy way to ensure we don't change anything visually while also fixing the accessibility issue. Using html will most likely mean having to also re-style the items.